data/data/bootstrap/files/usr/local/bin/wait-for-ha-api.sh

```diff
@@ -1,10 +1,24 @@
 #!/usr/bin/env bash
 
 wait_for_ha_api() {
-    if [ "$BOOTSTRAP_INPLACE" = true ]
-    then
-	return 0
-    fi
+    while :
```

Comment:
Mixing tabs and spaces makes this difficult to read.  Better to use spaces here to be consistent with the rest of the file.